engine/src/flutter/.ci.yaml

```diff
@@ -500,54 +500,18 @@ targets:
       - cpu=x86
 
   - name: Linux windows_android_aot_engine
-    recipe: engine_v2/engine_v2
```

Comment:
Are you deleting everything but the name because of the weird merged namespaces in ci.yaml we have with release builders?